src/popup/router/components/Modals/QrCodeReader.vue

```diff
@@ -190,5 +205,28 @@ export default {
       max-width: 100%;
     }
   }
+
+  .subtitle {
+    @extend %face-sans-14-bold;
+
+    color: variables.$color-light-grey;
+  }
+
+  .text {
+    @extend %face-sans-13-medium;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
 }
+
+.modal .container .footer .content .button {
```

Comment:
I suggest to not use the child component's class names to match the local elements.